nifi-nar-bundles/nifi-confluent-platform-bundle/nifi-confluent-schema-registry-service/src/main/java/org/apache/nifi/confluent/schemaregistry/ConfluentSchemaRegistry.java

```diff
@@ -166,6 +202,21 @@ public void onEnabled(final ConfigurationContext context) {
             }
         }
 
+        final boolean authTypeSet = validationContext.getProperty(PROP_AUTH_TYPE).isSet();
+        final boolean authUsernameSet = validationContext.getProperty(PROP_BASIC_AUTH_USERNAME).isSet();
+        final boolean authPasswordSet = validationContext.getProperty(PROP_BASIC_AUTH_PASSWORD).isSet();
```

Comment:
A password value of `    ` (4 space characters) for example would register as `isSet()` here but `StringUtils.trimToEmpty()` on line 165 above would return an empty string.

Reply:
Good point, as i suggested in the other comment i think I need a decision on whether we want to support null passwords, and then i can rework this for you.

Reply:
In the re-worked validator, I first trimToEmpty() and base my empty determination on the trimmed version.